arrow-optics/src/main/kotlin/arrow/optics/extensions/mapk/at/MapKAt.kt

```diff
@@ -0,0 +1,43 @@
+package arrow.optics.extensions.mapk.at
+
+import arrow.core.MapK
+import arrow.core.MapK.Companion
+import arrow.core.Option
+import arrow.optics.PLens
+import arrow.optics.extensions.MapKAt
+
+/**
+ * cached extension
+ */
+@PublishedApi()
+internal val at_singleton: MapKAt<Any?, Any?> = object : MapKAt<Any?, Any?> {}
+
+@JvmName("at")
+@Suppress(
+  "UNCHECKED_CAST",
+  "USELESS_CAST",
+  "EXTENSION_SHADOWED_BY_MEMBER",
+  "UNUSED_PARAMETER"
+)
+@Deprecated(
+  "arrow.optics.extensions package is being deprecated, and it will be removed in 0.13.",
+  level = DeprecationLevel.WARNING
+)
```

Comment:
```suggestion
@Deprecated(
  "arrow.optics.extensions package is being deprecated, and it will be removed in 0.13.",
  ReplaceWith(
    "At.map<K, V>().at(i)",
    "arrow.optics.map", "arrow.optics.typeclasses.At"
  ),
  level = DeprecationLevel.WARNING
)
```

Reply:
Same here